### PR TITLE
address open issue 12

### PR DESCRIPTION
**Pull request by timsully**
Hi Steve,

I hope all is well. I've adjusted and tested the `h1`, `h2`, and `p` tags on <= 640px width screens alongside some `%` based `line-height` values that I fixed. Let me know if anything else needs to be addressed.

All the best.

### CODE REVIEW

File: public/global.css [hunk 1]
```diff
@@ -233,3 +233,24 @@ button.secondary {
...
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+  }
+
+  p {
+    font-size: 16px;
+    line-height: 30px;
+  }
+}
```
Comment (elimisteve) [inline public/global.css:256]: @timsully This should be good :+1: . Be sure to do cache-busting on the URL to `global.css` in `index.html` whenever we update `global.css`.

Comment (timsully) [reply public/global.css:256]: Would I bump the version on the query string, like this? 

Example:
`<link rel="stylesheet" href="style.css?v=3.4.1">`

**Comment by elimisteve** [comment]
@timsully Yes, basically; see index.html and the top of globals.css for examples.

**Comment by timsully** [comment]
@elimisteve Addressed cache-busting.

**Comment by timsully** [comment]
Awesome, thank you for the context and noted!

**Comment by elimisteve** [comment]
@timsully Tweaked and merged! :tada:

**Comment by elimisteve** [comment]
Deployed, too :smile:

**Comment by elimisteve** [comment]
Good work, sir!

**Pull request merged**